scripts/ingests/beiler24/ingestsources_b24.py

```diff
@@ -114,13 +115,23 @@ def ingest_spectra(data):
             print("\n------------------")
 
         except AstroDBError as e:
-            skipped += 1
-            logger.info(f"Spectra skipped: {skipped}")
-            print(f"Error ingesting spectrum for {source_name}: {e}")
+            if "suspected duplicate" in str(e):
+                skipped += 1
+                logger.info(f"Spectra skipped: {skipped}")
+            elif "does not appear to be accessible" in str(e):
+                inaccessible += 1
+                logger.error(f"Inaccessible spectrum for {source_name}: {e}")
+            else:
+                logger.error(
+                    f"Unexpected error ingesting spectrum for {source_name}: {e}"
+                )
+                raise e
```

Comment:
if there's an unanticipated exception, you should raise the error. Otherwise, it just gets lost in the shuffle.